cloud_experts_tutorials/cloud-experts-dynamic-certificate-custom-domain.adoc

```diff
@@ -38,24 +38,35 @@ Learn how to use the link:https://docs.openshift.com/container-platform/latest/s
 +
 [source,terminal]
 ----
-$ export DOMAIN=apps.<company_name>.io <1>
-$ export EMAIL=<youremail@company_name.io> <2>
+$ export DOMAIN=apps.example.com <1>
+$ export EMAIL=email@example.com <2>
 $ export AWS_PAGER=""
-$ export CLUSTER_NAME=$(oc get infrastructure cluster -o=jsonpath="{.status.infrastructureName}"  | sed 's/-[a-z0-9]\{5\}$//')
+$ export CLUSTER=$(oc get infrastructure cluster -o=jsonpath="{.status.infrastructureName}"  | sed 's/-[a-z0-9]\{5\}$//')
 $ export OIDC_ENDPOINT=$(oc get authentication.config.openshift.io cluster -o json | jq -r .spec.serviceAccountIssuer | sed  's|^https://||')
 $ export REGION=$(oc get infrastructure cluster -o=jsonpath="{.status.platformStatus.aws.region}")
 $ export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
-$ export SCRATCH="/tmp/${CLUSTER_NAME}/dynamic-certs"
+$ export SCRATCH="/tmp/${CLUSTER}/dynamic-certs"
 $ mkdir -p ${SCRATCH}
 ----
-<1> The custom domain.
-<2> The e-mail Let's Encrypt will use to send notifications about your certificates.
+<1> Replace with the custom domain you want to use for the `IngressController`.
+<2> Replace with the e-mail you want Let's Encrypt to use to send notifications about your certificates.
++
 . Ensure all fields output correctly before moving to the next section:
 +
 [source,terminal]
 ----
-$ echo "Cluster: ${CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
+$ echo "Cluster: ${CLUSTER}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
 ----
++
+[NOTE]
+====
+The "Cluster" output from the above command may be the name of your cluster, the internal ID of your cluster, or the cluster's domain prefix. If you prefer to use another identifier, you can manually set this value by running the following command:
++
```

Comment:
~~Need to add `====` before the `+` on line 64:~~ You need to delete the `+` on line 64. The note should display that note with the correct spacing for the note's paragraph

![image](https://github.com/openshift/openshift-docs/assets/16167833/48987788-7cd2-4496-94e8-03b0aaaa8ec7)